src/Microsoft.Diagnostics.EventFlow.Core/Implementations/EventData.cs

```diff
@@ -28,6 +29,8 @@ public class EventData: IDeepCloneable<EventData>
 
         public long Keywords { get; set; }
 
+        public object EventContext { get; set; } = ContextResolver.GetRequestContext<object>();
```

Comment:
Context is now only filled in Trace and EventSource input. AppInsights output also need to restore context, so it depends on Correlation lib